FirebaseTestingSupport/README.md

```diff
@@ -0,0 +1,53 @@
+# Firebase Testing Support
```

Comment:
Clarify that these instructions are for test creators not test users.

Reply:
Updated, PTAL.